devtools/conda-envs/queued-submit.yaml

```diff
@@ -7,11 +7,13 @@ dependencies:
   - python
   - pip
   - qcportal >=0.49
-  - openff-qcsubmit >=0.50.0
+  #- openff-qcsubmit >=0.50.0
   - openff-toolkit-base >=0.12
   - openff-forcefields >=2.0.0
   - PyGithub
   - pandas
   - tabulate
   - importlib_metadata =5.0.0
   - openmm
+  - pip:
+    - git+https://github.com/openforcefield/openff-qcsubmit.git@0.50.1
```

Comment:
Info only (not a critique): these builds should be online soon, I'm just crossing the "i"s and dotting the "t"s:  https://github.com/conda-forge/openff-qcsubmit-feedstock/pull/15

Reply:
Cool, thank you! If they go up before we merge, I'll switch back to using the conda packages in these envs in this PR. Otherwise will do it post-hoc.

Reply:
Done! Thanks again @mattwthompson!